### PR TITLE
Show path to buildtest configuration file

### LINKED ISSUE #1099: [FEATURE]: Show path to buildtest configuration file
### Please describe your feature request

Right now we don't have a quick way to check path to configuration file used for buildtest.

One can specify path on command line `buildtest -c <config_path>`, the path can be set via `$BUILDTEST_CONFIGFILE`. Right now one needs to run `buildtest config validate`  which shows the file that is being validated

```
(buildtest)  ~/gitrepos/buildtest/ [devel] buildtest config validate
/global/u1/s/siddiq90/gitrepos/buildtest-nersc/config.yml is valid
```

For this feature we would like to have a subcommand just to show path to configuration file this could be `buildtest config path` which shows path regardless if file is valid. This option doesn't need any other options just running the command should resolve to path of configuration file.

- [x] Add a new subcommand via argparser in  https://github.com/buildtesters/buildtest/blob/devel/buildtest/cli/__init__.py
- [x] Add method and implement logic in https://github.com/buildtesters/buildtest/blob/devel/buildtest/cli/config.py
- [x] Complete bash completion https://github.com/buildtesters/buildtest/blob/devel/bash_completion.sh
- [x] Add regression test in https://github.com/buildtesters/buildtest/blob/devel/tests/cli/test_config.py
- [x] Add user documentation in https://github.com/buildtesters/buildtest/blob/devel/docs/features.rst
- [x] Update `buildtest help config` section https://github.com/buildtesters/buildtest/blob/devel/buildtest/cli/help.py 

### Post question in Slack

- [X] I agree that I posted my question in [slack](https://hpcbuildtest.slack.com/) before creating this issue

### Is there an existing issue

- [X] I confirm there is no existing [issue](https://github.com/buildtesters/buildtest/issues) for this issue

### PR DESCRIPTION
**Pull request by Xiangs18**
- [x] [Issue#1099](https://github.com/buildtesters/buildtest/issues/1099) Completed
- [x] A bug fixed detected in branch devel

### CODE REVIEW

File: buildtest/cli/config.py [hunk 1]
```diff
@@ -118,6 +121,11 @@ def validate_config(configuration):
     console.print(f"{configuration.file} is valid")
 
 
+def view_path(configuration):
+    """Display the path to configuration file regardless if file is valid"""
```
Comment (shahzebsiddiqui) [inline buildtest/cli/config.py]: Can you add the following in docstrings. I think this is mentioned above in several methods

```
    Args:
        configuration (buildtest.config.SiteConfiguration): An instance of SiteConfiguration class
```

File: buildtest/cli/config.py [hunk 2]
```diff
@@ -118,6 +121,11 @@ def validate_config(configuration):
...
+    console.print(f"Config path: {configuration.file}")
```
Comment (shahzebsiddiqui) [inline buildtest/cli/config.py]: I am thinking we just print the file name without any text. Could be useful if one wants to do something like
```
ls -l $(buildtest config path)
cat $(buildtest config path)
cat $(buildtest config path) | grep
```

File: docs/features.rst [hunk 3]
```diff
@@ -332,6 +332,13 @@ If you want to view buildtest configuration you can run ``buildtest config view`
 
 .. command-output:: buildtest config view
 
+Check path to buildtest configuration file (``buildtest config path``)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (shahzebsiddiqui) [inline docs/features.rst]: Can you please finish the section `~` to end of section title

**Comment by Xiangs18** [comment]
Hi @shahzebsiddiqui, this PR is now ready for review. Thank you for your feedback!

**Comment by shahzebsiddiqui** [comment]
@Xiangs18 thanks for your contribution. This looks really good. I am merging this in

**Pull request merged**